framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/deployment/KieServerDeployment.java

```diff
@@ -17,8 +17,31 @@
 
 import java.net.URL;
 
+/**
+ * Kie Server deplyoment representation in cloud.
+ */
 public interface KieServerDeployment extends Deployment {
+
+    /**
+     * Get route URL for Kie Server service (deployment).
```

Comment:
Just "Get URL for Kie Server service (deployment).", route is OpenShift specific thing.